### PR TITLE
Add request parameters tag to documented endpoints

### PR DESCRIPTION
**Pull request by lucaslago**
- [x] Use the custom tag created on #89 in our markdown files.
image

### CODE REVIEW

File: source/includes/telemetry_api/_devices.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Devices transmission rate
+# Devices transmission rate
```
Comment (waldemarnt) [inline source/includes/telemetry_api/_devices.md]: Why did you change that? this file is not been used, we moved to glossary

Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: Probably because the linter was complaining about using `##`s without a previous `#`. Since we are not using this file anymore can we remove it?

Comment (waldemarnt) [inline source/includes/telemetry_api/_devices.md]: Please, remove 💃

**Comment by waldemarnt** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:
